modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java

```diff
@@ -942,6 +1043,76 @@ private void _testPostAccountWithContactInformation() throws Exception {
 			postAccount.getAccountContactInformation());
 	}
 
+	private void _testPostAccountWithMoreExternalReferenceCodes()
+		throws Exception {
+
+		Account parentAccount = randomAccount();
+		Account randomAccount = randomAccount();
+
+		Account postParentAccount = testPostAccount_addAccount(parentAccount);
+
+		Organization organization1 = OrganizationTestUtil.addOrganization();
+		Organization organization2 = OrganizationTestUtil.addOrganization();
+		Address billingAddress = OrganizationTestUtil.addAddress(organization1);
+		Address shippingAddress = OrganizationTestUtil.addAddress(
+			organization2);
+
+		FileEntry fileEntry = _addFileEntry();
+
+		randomAccount.setDefaultBillingAddressExternalReferenceCode(
+			billingAddress.getExternalReferenceCode());
+		randomAccount.setDefaultBillingAddressId(0L);
+		randomAccount.setDefaultShippingAddressExternalReferenceCode(
+			shippingAddress.getExternalReferenceCode());
+		randomAccount.setDefaultShippingAddressId(0L);
+		randomAccount.setLogoExternalReferenceCode(
+			fileEntry.getExternalReferenceCode());
+		randomAccount.setLogoId(0L);
+		randomAccount.setOrganizationExternalReferenceCodes(
+			new String[] {
+				organization1.getExternalReferenceCode(),
+				organization2.getExternalReferenceCode()
+			});
+		randomAccount.setOrganizationIds(new Long[0]);
+		randomAccount.setParentAccountExternalReferenceCode(
+			postParentAccount.getExternalReferenceCode());
+		randomAccount.setParentAccountId(0L);
+
+		Account postAccount = testPostAccount_addAccount(randomAccount);
+
+		randomAccount.setDefaultBillingAddressId(billingAddress.getAddressId());
+		randomAccount.setDefaultShippingAddressId(
+			shippingAddress.getAddressId());
+		randomAccount.setLogoId(fileEntry.getFileEntryId());
+		randomAccount.setOrganizationIds(
+			new Long[] {
+				organization1.getOrganizationId(),
+				organization2.getOrganizationId()
+			});
+		randomAccount.setParentAccountId(postParentAccount.getId());
+
+		Arrays.sort(randomAccount.getOrganizationIds());
+		Arrays.sort(postAccount.getOrganizationIds());
+
+		Assert.assertEquals(
+			randomAccount.getDefaultBillingAddressId(),
+			postAccount.getDefaultBillingAddressId());
+		Assert.assertEquals(
+			randomAccount.getDefaultShippingAddressId(),
+			postAccount.getDefaultShippingAddressId());
+		//		Assert.assertEquals(
+		//			randomAccount.getLogoId(),
+		//			postAccount.getLogoId());
+		Assert.assertArrayEquals(
+			randomAccount.getOrganizationIds(),
+			postAccount.getOrganizationIds());
+		Assert.assertEquals(
+			randomAccount.getParentAccountId(),
+			postAccount.getParentAccountId());
+
+		assertValid(postAccount);
```

Comment:
move before or remove it (same as the path - if you moved in the previous one move also here, if you removed remove also here)